code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -872,12 +873,27 @@ var/global/list/belly_colorable_only_fullscreens = list("a_synth_flesh_mono",
 					b.absorb_living(ourtarget)
 				if("Cancel")
 					return
+		if("Health Check")
+			var/mob/living/carbon/human/H = target
+			var/target_health = round((H.health/H.getMaxHealth())*100)
+			to_chat(usr, "<span class= 'warning'>\The [target] is at [target_health]% health.</span>")
+			if(H.weakened)
+				to_chat(usr, "<span class= 'warning'>\The [target] is currently weakened, this does not stop them from hearing emotes or emoting themselves.</span>")
+			if(H.blinded && H.paralysis)
```

Comment:
A suggestion I have is to append the messages in something like.
The [target] is currently [list of effects]. This stops them from [list of consequences].

Reply:
Had a go at doing this. Got rid of the weakened one seeing as it doesn't affect their ability to emote anyway.